pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
```

Comment:
with subsections it works this way that the "starting weight" is the weight of the `_section.md` so change it to 50 or more and all the next documents should follow. This way you fix preview and make sure it shows in the UI